### DISCUSSION TITLE
404 page while logging into github (astro integration)

### DISCUSSION THREAD

**Discussion by eyrewiut** [Thinkmill/keystatic#360] [Q&A]
I have set up my site with keystatic and everything works fine, however there seems to be a glitch when logging into the cms.

When I click the "log in with github" button, it shows an unstyled 404 page as shown below, but if I reload the page it logs in and works fine thereafter.. I am trying to use the new "hybrid" rendering mode, but have also tried it with the standard SSR mode as well with the same result.

Does anyone have any idea what is causing this? 

URL: `/api/keystatic/github/login`
![screenshot of the 404 page displayed during login flow](https://github.com/Thinkmill/keystatic/assets/35190402/5c4c478a-265f-4b30-b8b2-0bee933d7d2d)

**Comment by simonswiss** [comment]
Hey!

I have also noticed this, and it always works after refreshing. I'll post an update here when I work out why this is happening 👍
